syft/pkg/cataloger/python/parse_wheel_egg.go

```diff
@@ -40,6 +40,7 @@ func parseWheelOrEgg(ctx context.Context, resolver file.Resolver, _ *generic.Env
 		newPackageForPackage(
 			*pd,
 			findLicenses(ctx, licenseScanner, resolver, *pd),
+			pkg.CompleteDependencies,
```

Comment:
I'm missing something. This says "complete dependencies" but always returns a nil relationships array.

Reply:
In cases where a cataloger parser function only has visibility about a single package, but the parser function is called numerous times in a filetree, then there needs to be a post-cataloger function that "ties" the discovered packages together outside of the parser function -- python egg/wheel metadata works like this. Here's the post-processor... invoked within the cataloger after all parser function calls, but before it returns it's results: https://github.com/anchore/syft/blob/05e8ba948d94de9a3df86c79bd0255eabefea1b8/syft/pkg/cataloger/python/cataloger.go#L43 .